index.js

```diff
@@ -1,17 +1,201 @@
+  popup.classList.add('popup');
+
+  const header = document.createElement('h2');
+  header.classList.add('work-title');
+  header.innerHTML = cardArray[elem].title;
+
+  const popFrame = document.createElement('div');
+  popFrame.classList.add('frame');
+  popFrame.innerHTML = frameInfo;
+
+  const popContainer = document.createElement('div');
+  popContainer.classList.add('pop-container');
+  const popimg = document.createElement('img');
+  popimg.setAttribute('src', cardArray[elem].img);
+  popContainer.appendChild(popimg);
+
+  const lang = document.createElement('ul');
+  lang.classList.add('tags');
+
+  cardArray[elem].tech.forEach((item) => {
+    const tagChild = document.createElement('li');
+    tagChild.innerHTML = item;
+    lang.appendChild(tagChild);
+  });
+
+  const txt = document.createElement('p');
+  txt.classList.add('pop-para');
+  txt.innerHTML = cardArray[elem].description;
+
+  const btnLive = document.createElement('button');
+  btnLive.classList.add('btn');
+  const icon = document.createElement('img');
+  icon.classList.add('span');
+  icon.setAttribute('src', 'icons/icon.png');
+  btnLive.innerHTML = 'see live';
+  btnLive.appendChild(icon);
+  const btnSrc = document.createElement('button');
+  btnSrc.classList.add('btn');
+  const gitIcon = document.createElement('img');
+  gitIcon.classList.add('span');
+  gitIcon.setAttribute('src', 'icons/icons8-github-48.png');
+  btnSrc.innerHTML = 'see source';
+  btnSrc.appendChild(gitIcon);
+  const btnContainer = document.createElement('div');
+  btnContainer.classList.add('btn-container');
+  btnContainer.append(btnLive, btnSrc);
+  const rightBlock = document.createElement('div');
+  rightBlock.classList.add('right-block');
+  rightBlock.append(lang, btnContainer);
+
+  const popLeftBlock = document.createElement('div');
+  const closeMark = document.createElement('div');
+  const dlt = document.createElement('img');
+  dlt.classList.add('xmark');
+  dlt.setAttribute('src', 'icons/dlt-96.png');
+  closeMark.appendChild(dlt);
+  popLeftBlock.classList.add('left-blockpop');
+  popLeftBlock.append(txt, rightBlock);
+  popup.append(closeMark, header, popFrame, popContainer, popLeftBlock);
+
+  leftBlock.append(title, frame, para, tags, buttonEl);
+
+  card.append(popup, imgContainer, leftBlock);
+  sectionEl.appendChild(card);
+
+  buttonEl.addEventListener('click', () => {
+    popup.classList.toggle('show');
+  });
+
+  dlt.addEventListener('click', () => {
+    popup.classList.toggle('show');
+  });
+});
+
+function showError(input, message) {
+  msgEl.innerHTML = message;
+}
+
+function showSuccess() {
+  formControl.classList.add('valid');
+}
+
+function checkInput() {
+  const emailValue = emailEl.value;
+
+  if (emailValue !== emailValue.toLowerCase()) {
+    showError(emailEl, 'The content has to be in lower case!');
+  } else {
+    showSuccess(emailEl);
+  }
+}
+
+formEl.addEventListener('submit', (e) => {
+  e.preventDefault();
+
+  checkInput();
+});
```

Comment:
- [ ] Good job implementing the error message when there is an uppercase in email, :clap: however your form is not submitting in all cases, kindly fix this as required by the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md#:~:text=When%20the%20user,form%20is%20sent.) :recycle: \